deploy/createTypesPackages.mjs

```diff
@@ -131,3 +129,7 @@ function copyREADME(pkg, pkgJSON, writePath) {
 
   fs.writeFileSync(writePath, readme);
 }
+
+if (process.argv[1] === fileURLToPath(import.meta.url)) {
+  go();
```

Comment:
```suggestion
  await go();
```

Using top level `await` can be helpful as it changes exit code when throws. [It's supported by Node.js 14+.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await#browser_compatibility)